src/Elastic.Apm/Model/Transaction.cs

```diff
@@ -27,6 +27,10 @@ internal class Transaction : ITransaction
 {
 	internal static readonly string ApmTransactionActivityName = "ElasticApm.Transaction";
 
+#if NET
+	internal static readonly ActivitySource ElasticApmActivitySource = new("Elastic.Apm");
```

Comment:
👍

Reply:
Should we bump our DiagnosticSource reference from 5.0.0? 

We can do in a follow up PR.

Reply:
Yeah, I'll open an issue to track that if we don't have one. I think we should, but I'm not sure about the implications, so we'll need to spike it and see if anything breaks!